xml/System.Formats.Cbor/CborContentException.xml

```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>The exception that is thrown when Concise Binary Object Representation (CBOR) content is invalid.</summary>
```

Comment:
Do we need to do this for every type? It feels redundant since the namespace also contains the acronym. I would suggest expanding it only for the CborWriter and CborReader classes, since they are the entrypoints to this particular API.